.github/workflows/pipeline-artefacts-cleardown.yaml

```diff
@@ -0,0 +1,45 @@
+name: Pipeline Cleardown Artefacts
+
+on:
+  delete:
+    branches:
+      - 'task/*'
+jobs:
+  metadata:
+    if: github.event.ref_type == 'branch' && startswith(github.event.ref, 'task/') && github.actor != 'github-merge-queue[bot]'
+    name: "Get Metadata"
+    uses: NHSDigital/uec-dos-management/.github/workflows/metadata.yaml@v0.25
+  cleardown-artefacts:
```

Comment:
Do we need this new workflow triggered on delete ? Could we not add the cleardown-artefacts job to the existing pipeline-infrastructure-cleardown-dev pipeline

Reply:
plugged cleardown-artefacts job into existing pipeline-infrastructure-cleardown-dev pipeline per suggestion